src/styles/annotator/components/NotebookModal.scss

```diff
@@ -0,0 +1,50 @@
+@use '../variables' as var;
```

Comment:
I presume no changes, just moving, correct?

Reply:
Mostly moving. In addition:

- renamed the classes `Notebook__` to `NotebootkModal__` (that's why I needed to fix the broken tests)
- used `&__` syntax in the scss
- made `NotebookModalIframe` class